net-misc/networkmanager-fortisslvpn/networkmanager-fortisslvpn-1.2.2.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+GNOME_ORG_MODULE="NetworkManager-${PN##*-}"
+
+inherit gnome2 autotools
+
+DESCRIPTION="NetworkManager Fortinet SSLVPN compatible plugin"
+HOMEPAGE="https://wiki.gnome.org/Projects/NetworkManager"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="gtk"
+
+RDEPEND="
+	>=net-misc/networkmanager-1.2.2:=
+	>=dev-libs/dbus-glib-0.74
+	>=dev-libs/glib-2.32:2
+	>=net-misc/openfortivpn-1.2.0
+	gtk? (
+		app-crypt/libsecret
+		>=gnome-extra/nm-applet-1.0.5
+		>=x11-libs/gtk+-3.4:3
+	)
+"
+DEPEND="${RDEPEND}
+	sys-devel/gettext
+	dev-util/intltool
+	virtual/pkgconfig
+"
+
+PATCHES=(
+	"${FILESDIR}/${P}-location-fix.diff"
+)
+
+src_prepare() {
+	gnome2_src_prepare
+
+	AT_NOELIBTOOLIZE=yes eautoreconf
```

Comment:
As discussed on IRC in #gentoo-desktop this morning, `gnome2_src_prepare` must be called after eautoreconf as it calls functions that might modify configure (and not configure.ac/in due to triggering maintainer-mode). Not calling `gnome2_src_prepare` leaves you exposed to various phases of the build accessing files outside of `${WORKDIR}` and so is not recommended, especially for packages using glib, dbus or gobject-introspection.